packages/ui/src/components/Badge/Badge.vue

```diff
@@ -0,0 +1,117 @@
+<script setup lang="ts">
+import {SSpinner} from '../Spinner'
+
+type BdgType = 'active' | 'error' | 'warning' | 'info' | 'debug' | 'pending'
+
+const props = withDefaults(
+  defineProps<{
+    type?: BdgType
+    colorBackground?: boolean
+    withBorder?: boolean
+    onlyMarker?: boolean
+  }>(),
+  {
+    type: 'active',
+    colorBackground: false,
+    withBorder: false,
+    onlyMarker: false
+  },
+)
+
+const showSpinner = computed(() => {
+  return props.type === 'pending'
+})
+
+</script>
+
+<template>
+  <div
+    tabindex="0"
+    class=""
+    :class="[
+      's-badge',
+      {'s-border' : withBorder},

```

Comment:
I am a bit confused about `s-border` class. It is specific for Badge component, so, according to BEM, it should be named as `s-badge--border` or `s-badge--bordered` or something. But in this case it is OK because of `scoped` style. But if it is scoped, why to use BEM at all?
Also I am not sure about usability of `scoped` in terms of component library, especially when it is about such a small component as Badge one which is the best pretender to be refactored as functional component (for performance reasons) in future. `scoped` style doesn't work with functional components. So, it is my point to not use `scoped` with components that are almost stateless.
Anyway, I am just confused. That's not a critical note, but if I were you, I would like to drop `scoped` and transform `s-border` -> `s-badge--border`.